src/main/java/io/github/jazorp/Validators.java

```diff
@@ -63,5 +63,21 @@ public static ValidationThunk length(String field, String value, int length) {
     public static ValidationThunk email(String field, String value) {
         return validateImpl(v -> EMAIL_PATTERN.matcher(value).matches(), ErrorType.EMAIL, field, value);
     }
+
+    public static ValidationThunk lt(String field, Number value, Number max) {
+        return validateImpl(v -> v.doubleValue() < max.doubleValue(), ErrorType.LT, field, value, max);
+    }
+
+    public static ValidationThunk lte(String field, Number value, Number max) {
+        return validateImpl(v -> v.doubleValue() <= max.doubleValue(), ErrorType.LTE, field, value, max);
+    }
+
+    public static ValidationThunk gt(String field, Number value, Number min) {
+        return validateImpl(v -> v.doubleValue() > min.doubleValue(), ErrorType.GT, field, value, min);
+    }
+
+    public static ValidationThunk gte(String field, Number value, Number min) {
+        return validateImpl(v -> v.doubleValue() >= min.doubleValue(), ErrorType.GTE, field, value, min);
```

Comment:
As already mentioned in the ticket, avoid doing exact double comparisons. See [this](https://stackoverflow.com/questions/8081827/how-to-compare-two-double-values-in-java) SO post.

Reply:
would you like to have the delta passed the full way?

```
Validators.lt("foo", 5.001, 5.002, 0.001)
```
or is it ok to define the delta hardcoded, if hardcoded which delta value to you prefer?

Reply:
Check [this](https://stackoverflow.com/questions/6837007/comparing-float-double-values-using-operator). Let's have this value by default but allow the user to override it.

Not entirely sure how this play out for ints, doubles and floats at the same time.